lib/ansible/modules/network/nxos/nxos_config.py

```diff
@@ -390,13 +390,16 @@ def main():
     running_config = None
     startup_config = None
 
+    sh_run_config = {'command': 'show running-config', 'output': 'text'}
```

Comment:
Instead of hard coding we can actually do something like:
```
def execute_show_command(command, module, output='text'):
    cmds = [{
        'command': command,
        'output': output,
    }]
    body = run_commands(module, cmds)
    return body
```